src/reporter/tests/test_NTNENA.py

```diff
@@ -30,7 +31,7 @@ def query(values=False, params=None, service=None):
     return requests.get(query_url(values), params=params, headers=h)
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope='function')
```

Comment:
changing from `module` to `function` increase a lot the testing time.

probably the best solution is to create a specific test file only using `reporter_dataset_different_types` as fixture.

Reply:
sure, I'll update it.

Reply:
done